nav2_safety_nodes/src/nav2_safety_node.cpp

```diff
@@ -0,0 +1,250 @@
+#include <chrono>
```

Comment:
Package needs a readme file with some basic information like
- What is this
- What does it do
- What are the parameters
- How does it work
- Any user advice required

This is a good example https://github.com/ros-planning/navigation2/tree/main/nav2_smac_planner though its much, much longer than what you need to do here. This is considerably simpler.